lib/pbench/server/cache_manager.py

```diff
@@ -1647,17 +1647,19 @@ def reached_goal():
                 reclaim_failed += 1
                 error = e
             if error:
-                self.logger.error("RECLAIM {} failed with '{}'", name, error)
-        reached = reached_goal()
+                self.logger.error("RECLAIM: {} failed with '{}'", name, error)
+        goal_check = reached_goal()
+        free_pct = goal_check.usage.free * 100.0 / goal_check.usage.total
         self.logger.info(
-            "RECLAIM summary: goal {}%, {} {}: "
-            "{} datasets, {} had cache: {} reclaimed and {} errors",
+            "RECLAIM (goal {}%, {}) {}: {} datasets, "
+            "{} had cache: {} reclaimed and {} errors: {:.1f}% free",
             goal_pct,
             humanize.naturalsize(goal_bytes),
-            "achieved" if reached else "failed",
+            "achieved" if goal_check.reached else "failed",
```

Comment:
The added parenthesis doesn't quite work with the phrasing.  That is, the message used to be, "goal 20%, 1M achieved"; now it's "RECLAIM (...) achieved"...which is sorta OK, but it seems to me that "RECLAIM" is the name of the tool, and it was the _goal_ which was achieved (or failed).  You can fix the problem (if you're so inclined...) by moving the "achieved"/"failed" argument to before the parens.

Reply:
The format didn't bother me, but on consideration (and especially with respect to ops review) I decided that the "failed" did ... it's not really a "failure", we just weren't able to free as much space as requested. But rather than make the wording more complicated I just changed that to "partial", and shuffled things around a bit.